src/ClimaAtmos.jl

```diff
@@ -110,6 +110,8 @@ include(joinpath("prognostic_equations", "limited_tendencies.jl"))
 
 include(joinpath("callbacks", "callbacks.jl"))
 
+include(joinpath("diagnostics", "Diagnostics.jl"))
+
```

Comment:
```suggestion
import .Diagnostics as CAD
```
I might suggest putting this import here